Sources/OpenTelemetrySdk/Metrics/Export/PushMetricController.swift

```diff
@@ -6,35 +6,45 @@ import Foundation
 
 class PushMetricController {
     var meterSharedState: MeterSharedState
-    var meterProvider: MeterProviderSdk
+    unowned var meterProvider: MeterProviderSdk
```

Comment:
```suggestion
    weak var meterProvider: MeterProviderSdk?
```
Being unowned meterProvider could be destroyed while reading the values and the app would crash, being `weak` allows us to retain the instance in the timer event